### PR TITLE
chore(tree, engine, prune, stages, storage): improve logs

### PR DESCRIPTION
**Pull request by shekhirin**
1. Move all `tracing::instrument` spans of blockchain tree to `TRACE` level
2. Add chain canonicalization event and log it with `INFO` level in `bin/reth/src/node/events.rs` instead of blockchain tree
3. Add pruner events and log them with `INFO` level in `bin/reth/src/node/events.rs`
4. Move Execution stage and EVM time logs to `DEBUG` level

### CODE REVIEW

File: bin/reth/src/node/events.rs [hunk 1]
```diff
@@ -127,6 +128,14 @@ impl NodeState {
 
                 info!(number=block.number, hash=?block.hash, "Block added to canonical chain");
             }
+            BeaconConsensusEngineEvent::ChainCanonicalized(outcome, elapsed) => match outcome {
+                CanonicalOutcome::AlreadyCanonical { header } => {
+                    info!(number=header.number, header=?header.hash, ?elapsed, "Chain is already canonical");
```
Comment (rkrasiuk) [inline bin/reth/src/node/events.rs:133]: should this be a warning?

**Review by rkrasiuk — APPROVED** [review]
lgtm

File: bin/reth/src/node/events.rs (hunk 1 above)
Comment (mattsse) [reply bin/reth/src/node/events.rs:133]: this commonly happens on start up and could also happen if there are more CL connected

info is fine here imo

**Review by mattsse — APPROVED** [review]
seems reasonable to me

File: bin/reth/src/node/events.rs (hunk 1 above)
Comment (rkrasiuk) [reply bin/reth/src/node/events.rs:133]: i'd prefer downgrading it to trace/debug level, but no strong opinion on this

Comment (shekhirin) [reply bin/reth/src/node/events.rs:133]: agree on debug level, because it might be seem like something's wrong while in reality it was a no-op

**Pull request merged**